app/src/main/java/com/gh4a/ServiceFactory.java

```diff
@@ -259,48 +243,15 @@ public static OkHttpClient getImageHttpClient() {
     }
 
     static void initClient(Context context) {
-        sApiHttpClient = enableTls12IfNeeded(new OkHttpClient.Builder())
-                .cache(new Cache(new File(context.getCacheDir(), "api-http"), 20 * 1024 * 1024))
+        int twentyMB = 20 * 1024 * 1024;
+        sApiHttpClient = new OkHttpClient.Builder()
+                .cache(new Cache(new File(context.getCacheDir(), "api-http"), twentyMB))
                 .build();
-        sImageHttpClient = new OkHttpClient.Builder()
-                .cache(new Cache(new File(context.getCacheDir(), "image-http"), 20 * 1024 * 1024))
+        sImageHttpClient = sApiHttpClient.newBuilder()
```

Comment:
This isn't strictly neccesary, but will maximise reuse of functionality across clients while having separate cache still.  Generally best practice in OkHttp is to start with a single singleton client and fork things from there.

Main reason to not do this is if you observe slow and large image fetches delaying API traffic, but with HTTP/2, hopefully that's not happening routinely. That scenario is more likely mixing video streaming etc.